test/FastTests/NewClient/Query.cs

```diff
@@ -55,81 +55,6 @@ public void Query_With_Where_Clause()
                 }
             }
         }
-
-        [Fact]
-        public void Query_By_Index()
```

Comment:
So why remove it? If this is a temporary condition, then skip it until we can recover